website/docs/Software/OpenBCISoftware/01-OpenBCI_GUI.md

```diff
@@ -319,8 +319,69 @@ If you are encountering issues launching the GUI at this point, please head to t
+| 16     | Other                   | See below                                                |
+| 17     | Other                   | See below                                                |
+| 18     | Other                   | See below                                                |
+| 19     | Other                   | See below                                                |
+| 20     | Analog Channel 0        | Analog channel 0                                         |
+| 21     | Analog Channel 1        | Analog channel 1                                         |
+| 22     | Analog Channel 2        | Analog channel 2                                         |
+| 23     | Timestamp               | Unix timestamp                                           |
+| 24     | Marker Channel          | Channel for adding manual markers to data                |
+| 25     | Timestamp (Formatted)   | Year-Month-Day Hour:Minute:Second (Not in Brainflow csv) |
+
+:::info Cyton Other Channels
 When recording using Cyton, the GUI will output a data file which contains 7 channels with the header "Other". These channels are user defined based on the [board mode](../../Cyton/04-OpenBCI_Cyton_SDK.md/#board-mode). Refer to the [aux data footer section](../../Cyton/03-Cyton_Data_Format.md/#firmware-version-200-fall-2016-to-now-1) of the Cyton data format documentation for further information.
 
-In digital read mode the 5th "Other" channel is connected to the D17 pin by default. This pin can be triggered by pressing the "PROG" button. This is useful for adding manual timestamps to your data.
\ No newline at end of file
+In digital read mode the 5th "Other" channel is connected to the D17 pin by default. This pin can be triggered by pressing the "PROG" button. This is useful for adding manual timestamps to your data.
+:::
+
+### Ganglion
+
+Here is a table of the column headers and descriptions for the Ganglion.
+
+| Column | OpenBCI-RAW             | Description |
+| ------ | ----------------------- | ------------- |
+| 1      | Sample Index            | The index of the sample per second (0-250)               |
+| 2      | EXG Channel 0           | EEG/EMG/ECG channel connected to pin labeled N1P         |
+| 3      | EXG Channel 1           | EEG/EMG/ECG channel connected to pin labeled N2P         |
+| 4      | EXG Channel 2           | EEG/EMG/ECG channel connected to pin labeled N3P         |
+| 5      | EXG Channel 3           | EEG/EMG/ECG channel connected to pin labeled N4P         |
+| 6      | Accel Channel 0         | Accelerometer channel 0 (X)                              |
+| 7      | Accel Channel 1         | Accelerometer channel 1 (Y)                              |
+| 8      | Accel Channel 2         | Accelerometer channel 2 (Z)                              |
+| 9      | Resistance Channel 0    | Impedence measurement for channel 0                      |
```

Comment:
I don't think this is accurate. We do not currently have a setup to record impedances to file. If implemented, it would be in a different file.

Reply:
These channels are not used in raw data files. Confirmed with testing Ganglion impedances while streaming (which really shouldn't be done) and reviewing related code.

Reply:
Sent you a message on Slack about this after a deep dive into what happens and why the impedance data never makes it to a raw data file. Saving impedance measurements to a separate file is still a meaningful task for the GUI in the future.